dev_guide/pod_autoscaling.adoc

```diff
@@ -0,0 +1,111 @@
+= Pod Autoscaling
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+A *HorizontalPodAutoscaler* object specifies how the system should automatically
+increase or decrease the scale of a *ReplicationController* or *DeploymentConfig*,
+based on metrics collected from the pods that belong to that *ReplicationController*
+or *DeploymentConfig*.
+
+== Requirements for Using Horizontal Pod Autoscalers
+
+In order to use *HorizontalPodAutoscalers*, your cluster admin must have
+link:../admin_guide/cluster_metrics.html[set up Heapster for cluster metrics],
+running in the `openshift-infra` namespace.
+
+== Metrics Supported
+
+The following describes the metrics supported by *HorizontalPodAutoscalers*:
+
+.Metrics
+[cols="3a,8a",options="header"]
+|===
+
+|Metric |Description
+
+|CPU Utilization
+|Percentage of the link:../dev_guide/compute_resources.html#cpu-requests[requested CPU]
+|===
+
+== Scaling
+
+Once a *HorizontalPodAutoscaler* is created, it will begin attempting to query
+Heapster for metrics on the pods.  It may take 1-2 minutes before Heapster obtains
+the initial metrics.  Once metrics are available in heapster, the *HorizontalPodAutoscaler*
+will compute the ratio of the current metric utilization with the desired metric utilitization,
+and scale up or down accordingly.  The scaling will occur at a regular interval, but it may take
+1-2 minutes before metrics make their way into Heapster.
+
+For *ReplicationControllers*, this scaling corresponds directly to the replicas of the
+*ReplicationController*.  For *DeploymentConfigs*, this will scale the latest deployment,
+or, if no deployment is present, the *ReplicationController* template in the *DeploymentConfig*
+(note that scaling will occur if the latest deployment is marked as `Complete`).
+
+== Sample Horizontal Pod Autoscaler File
+
+scaler.yaml
```

Comment:
Needs a `.` (no space) at the beginning if this was meant to be a title for the example file/block.